DOCKER.md

```diff
@@ -13,3 +13,17 @@ ADMIN_EMAILS=
 SENDGRID_USERNAME=
 SENDGRID_PASSWORD=
 ```
+
+If you would like to use [Amazon SES](https://aws.amazon.com/ses/) instead to send emails, you'll need a different set of environmental variables.
+
+```sh
+DATABASE_URL=
+SECRET_KEY_BASE=
+ADMIN_EMAILS=
```

Comment:
These top three are unrelated to the different email provider, no? I'm a bit confused by their inclusion here.